tutorials/lander.md

```diff
@@ -0,0 +1,422 @@
+
+Let's now compute the reference volume that will make the lander neutrally
+buoyant.
+
+$$volume\\_neutral = \frac{mass}{waterDensity} = \frac{1209.175}{1025} = 1.1796829268292683$$
+
+Now, let's account for the volume that our lander already has thanks to the
+low-volume outer collision elements:
+
+$$outer\\_volume = 2*1.1325*1.1503*0.000001 + 2*0.000001*1.1503*1.5 + 2*1.325* 0.000001*1.5 = 1.00313295e-05$$
+
+Let's now compute the volume of the main buoyancy `<collision>` element:
+
+$$main\\_buoyancy\\_volume = 1.1796829268292683 - 1.00313295e-05 = 1.1796728954997684$$
+
+Let's verify that our lander is neutrally buoyant now. Add the following
+collision element to your `base_link`:
+
+```xml
+<!-- Center of buoyancy -->
+<collision name="buoyancy">
+  <pose>0 0 0 0 0 0</pose>
+  <geometry>
+    <box>
+      <size>1.1796728954997684 1 1</size>
+    </box>
+  </geometry>
+</collision>
+```
+
+And launch Gazebo:
+
+```bash
+gz sim ~/gazebo_maritime/worlds/buoyant_lander.sdf
+```
+
+Hit play, and your lander should now maintain depth!
+
+This an example to illustrate how to calculate this reference value, however a 
+lander is normally designed to sink.
+
+Change the buoyancy plugin to the following value to be more realistic:
+
+```xml
+<!-- Center of buoyancy -->
+<collision name="buoyancy">
+  <pose>0 0 0 0 0 0</pose>
+  <geometry>
+    <box>
+      <size>1.14 1 1</size>
+    </box>
+  </geometry>
+</collision>
+```
+
+And launch Gazebo:
+
+```bash
+gz sim ~/gazebo_maritime/worlds/buoyant_lander.sdf
+```
+
+Hit play and the lander will slowly sink and hit the seafloor.
+
+### Hydrodynamics
+
+As an underwater vehicle, the lander should be influenced by the water as it
+moves. We can do that by attaching the `hydrodynamics` plugin to our lander.
+Add the next SDF block to your `model.sdf`. Note that we have empirically
+adjusted its values. Follow the
+[hydrodynamics tutorial](https://gazebosim.org/api/sim/8/theory_hydrodynamics.html)
+for recommendations about how to tune its values.
+
+```xml
+<!-- Hydrodynamics -->    
+<plugin
+  filename="gz-sim-hydrodynamics-system"
+  name="gz::sim::systems::Hydrodynamics">
+  <link_name>base_link</link_name>
+  <xDotU>-4.876161</xDotU>
+  <yDotV>-126.324739</yDotV>
+  <zDotW>-126.324739</zDotW>
+  <kDotP>0</kDotP>
+  <mDotQ>-33.46</mDotQ>
+  <nDotR>-33.46</nDotR>
+  <xUabsU>-6.2282</xUabsU>
+  <xU>0</xU>
+  <yVabsV>-601.27</yVabsV>
+  <yV>0</yV>
+  <zWabsW>-601.27</zWabsW>
+  <zW>0</zW>
+  <kPabsP>-0.1916</kPabsP>
+  <kP>0</kP>
+  <mQabsQ>-632.698957</mQabsQ>
+  <mQ>0</mQ>
+  <nRabsR>-632.698957</nRabsR>
+  <nR>0</nR>
+</plugin>
+```
+
+### Drop weight
+
+A common feature in maritime landers is to include a drop weight device. When
+this device is enabled, some part of the vehicle is dettached from the main
+chassis. At this point, the vehicle becomes positive buoyant, and starts moving
+up towards the water surface.
+
+In order to simulate this behavior, we'll need to make two changes to our lander
```

Comment:
some places use `behaviour` while some `behavior`. Keep them consistent?

Reply:
Updated in [f667c7e](https://github.com/gazebosim/gz-sim/pull/2259/commits/f667c7e279672cee3b39b5b17fc4162ed20a5929).